demdata-cms/package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```

Comment:
**Unexpected.** Is this file intentional?

Reply:
No, I'll remove this file.